src/main/mainWindow.ts

```diff
@@ -455,12 +455,20 @@ function createMainWindow() {
 
     win.webContents.setUserAgent(BrowserUserAgent);
 
-    const subdomain =
-        Settings.store.discordBranch === "canary" || Settings.store.discordBranch === "ptb"
-            ? `${Settings.store.discordBranch}.`
-            : "";
+    const branch = Settings.store.discordBranch;
+    const subdomain = branch === "canary" || branch === "ptb" ? `${branch}.` : "";
+    const uri = process.argv.find(arg => arg.startsWith("discord://"));
 
-    win.loadURL(`https://${subdomain}discord.com/app`);
+    const loadUrl = (url: string | undefined) => {
+        win.loadURL(`https://${subdomain}discord.com${url?.substring("discord://".length) || "/app"}`);
```

Comment:
Do not assume the host is an empty string, often times Discord URIs are `discord://-/`